pygeo/DVGeometry.py

```diff
@@ -1246,6 +1338,13 @@ def updateCalculations(self, new_pts, isComplex, config):
                 D[0] *= scale_x
                 D[1] *= scale_y
                 D[2] *= scale_z
+                if ang:
+                    # rotate non-aligned FFDs back to initial position
+                    D_    = numpy.copy(D)
+                    bp_rot = numpy.copy(base_pt) # here base_pt has been rotated
+                    D       = geo_utils.rotVbyW(D_ , ax_dir, -ang)
+                    base_pt = geo_utils.rotVbyW(bp_rot, ax_dir, -ang)
```

Comment:
I am 99% sure it's fine to do this but only if the rotation is truly implemented as a linear operator as it should be

Reply:
This is a good point. This came directly from the Mads/Sandy fix so I did not 100% thought it through, I will check the `rotVbyW` function.

Reply:
[The  function](https://github.com/mdolab/pygeo/blob/79fd7e7d87a4cd48a3586dc980262351823619e2/pygeo/geo_utils.py#L56-L85) ... do you think we should enforce some normalization of the rotation axis?